src/users/entities/user.entity.ts

```
@@ -1,44 +1,35 @@
+    @Column()
+    nickname: string;
+
+    @Column({
+      name: 'profile_image_name',
+      type: 'enum',
+      enum: ProfileImageName,
+      nullable: false
+    })
+    profileImageName: ProfileImageName;
+
+    @Column({ name: 'device_id' })
```

ESLint

[error] Delete `··`

(prettier/prettier)